build/ci.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash -ev
```

Comment:
run shellcheck on this? nothing serious in the output, but might as well :)

Reply:
Added in the travis.yaml